HW8_1.ipynb

```diff
@@ -0,0 +1,1611 @@
+{
```

Comment:
<p>Maybe a comment that this is the integral would be useful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>